subsystem/kubeapi_test.go

```diff
@@ -204,6 +221,6 @@ var _ = Describe("[kube-api]cluster installation", func() {
 		}
 		generateFullMeshConnectivity(ctx, "1.2.3.10", hosts...)
 
-		waitForClusterCRDState(ctx, kubeClient, key, models.ClusterStatusPreparingForInstallation, waitForClusterReconcileTimeout)
+		waitForClusterDeploymentCRDState(ctx, kubeClient, key, models.ClusterStatusReady, waitForClusterReconcileTimeout)
```

Comment:
@RazRegev let me know if changing to `ready` makes sense

Reply:
It's not. need to be `ClusterStatusPreparingForInstallation`